test/test_convergence.jl

```diff
@@ -153,48 +191,40 @@ end
 
 @testset "Check convergence (dry thermodynamic states): Stable/Unstable" begin
     for uf_type in [UF.BusingerType(), UF.GryanikType(), UF.GrachevType()]
-        for FT in [Float32, Float64]
+      for FT in [Float32,]
             profiles_sfc, profiles_int = generate_profiles(FT)
             scheme = [SF.FVScheme(), SF.FDScheme()]
-            z0_momentum = Array{FT}(range(1e-6, stop = 1e-1, length = 10))
-            z0_thermal = Array{FT}(range(1e-6, stop = 1e-1, length = 10))
-            maxiter = 10
-            check_over_dry_states(param_set, FT, profiles_int, profiles_sfc, scheme, z0_momentum, z0_thermal, maxiter)
+            z0_momentum = Array{FT}(range(1e-6, stop = 1e-1, length = 5))
+            z0_thermal = Array{FT}(range(1e-6, stop = 1e-1, length = 5))
+            maxiter = 40
+            tol_neutral = FT(SFP.cp_d(param_set) / 100)
+            for iteration_option in [false, ]
+              check_over_dry_states(
+                  param_set,
+                  FT,
+                  profiles_int,
+                  profiles_sfc,
+                  scheme,
+                  z0_momentum,
+                  z0_thermal,
+                  maxiter,
+                  tol_neutral,
+                  iteration_option,
+              )
+  #            check_over_moist_states(
```

Comment:
Is something failing?

Reply:
Nope - was testing the issues with buoyancy fluxes in dry, then moist cases - updated now.